scheduler-server/scheduling-dsl-compiler/src/libs/README

```diff
@@ -0,0 +1,2 @@
+Files in the ./constraints directory are copied from the constraints-dsl-compiler using a gradle action.
+These files should not be edited, or committed to version control.
```

Comment:
No obligation to make this change but it may be nice to make this `README.md`. That way a `find . -name \*.md` can return any docs in the repo :)

Reply:
Done